definitions/region/model_native_regions/euro-calliope_2.0.yaml

```diff
@@ -0,0 +1,36 @@
+- Euro-Calliope 2.0:
```

Comment:
Please add a comment with a reference to the model documentation, in particular the region list.
See this example: https://github.com/openENTRANCE/openentrance/blob/main/definitions/region/model_native_regions/remind_2.1.yaml